tests/10-vcdm2.js

```diff
@@ -691,8 +696,11 @@ describe('Securing Mechanisms', function() {
         'documents it produces using a securing mechanism as described in ' +
         'Section 4.9 Securing Mechanisms.', async function() {
         this.test.link = `https://w3c.github.io/vc-data-model/#securing-mechanisms:~:text=A%20conforming%20issuer%20implementation%20produces%20conforming%20documents%2C%20MUST%20include%20all%20required%20properties%20in%20the%20conforming%20documents%20that%20it%20produces%2C%20and%20MUST%20secure%20the%20conforming%20documents%20it%20produces%20using%20a%20securing%20mechanism%20as%20described%20in%20Section%204.9%20Securing%20Mechanisms.`;
-        should.exist(issuedVc, `Expected ${name} to issue a VC.`);
-        should.exist(issuedVc.proof, 'Expected VC to have a proof.');
+        // TODO: check "all required properties" (use a shared function)
+        issuedVc.should.have.property('type').that.does
+          .include('VerifiableCredential', `Expected ${name} to issue a VC.`);
```

Comment:
the assertion message here is kind of weird, but I do like the chains.

Reply:
Yeah...it's a copy paste from elsewhere (old code). I may update it at some point. Merging for now, though.